transformers/socket.io/client.js

```diff
@@ -22,15 +22,15 @@ module.exports = function client() {
   var factory = (function factory() {
     if ('undefined' !== typeof io && io.Socket) return io;
 
-    try { return Primus.require('socket.io-client'); }
+    try { return Primus.require('primus-socket.io-client'); }
```

Comment:
Should we add `socket.io-client` back in the devDependencies?

I also wonder what to do with the browser. I have removed all the patches (it was only one actually) and right now, we don't have a way to fallback.
The ability to opt-out for the fixes would be only for the node client.